snippets/ava.sublime-snippet

```diff
@@ -1,9 +1,9 @@
 <snippet>
     <content><![CDATA[
 import test from 'ava';
-import $1 from '$2';
+import $1 from ${2:'${1/([A-Z])/-\l$1/g}'};
 
-test('${3:title}', t => {
+test(${3/(.+)|.*/?1:'/}${3:title}${3/(.+)|.*/?1:', /}t => {
```

Comment:
Is the `|.*` part really needed? If yes, what purpose does it have? I might be missing something obvious since it's late here, but seems to be working fine without.